jina/types/document/uid.py

```diff
@@ -45,6 +45,7 @@ def get_content_hash(doc: 'DocumentProto') -> str:
     doc_without_id = DocumentProto()
     doc_without_id.CopyFrom(doc)
     doc_without_id.id = ""
+    del doc_without_id.chunks[:]
```

Comment:
This method should be overriden for MultiModalDocument where the chunks contents are considered for id. it can be done later

Reply:
I thought the id should be only from the content itself not the chunks?

Reply:
I'll open an issue as a reminder. I'll also mark it as 'good first issue', maybe someone will pick it up before we get to it ;)